tests/test_operator.py

```diff
@@ -1,1595 +0,0 @@
-import time
```

Comment:
i don't see tests/e2e/tests_operator.py in this PR

Reply:
https://github.com/zvonand/clickhouse-operator/blob/wrap_tests/tests/e2e/test_operator.py